test/protocols/autoTestGAPResults.sh

```diff
@@ -23,41 +23,41 @@ module list
 #1 Compress  and indexing the VCFfile
 
 
-for i in  $(ls "/groups/umcg-gsad/tmp01/projects/NIST_TRIO/run01/results/vcf/"*".vcf")
+for i in  $(ls "/groups/umcg-gsad/tmp01/projects/GAP/NIST_TRIO/run01/results/vcf/"*".vcf")
```

Comment:
Looks like a lot of paths start with the same `/groups/umcg-gsad/tmp01/projects/GAP/NIST_TRIO` and that would need to be changed again if you want to run the tests on another cluster/site; Would it not be better to define 1 variable for that path and use that instead?